lib/errors/spec.js

```diff
@@ -74,6 +74,21 @@ module.exports = [{
       name: 'invalidName',
       message: 'Invalid Name'
     }]
+  }, {
+    name: 'Trigger',
+    message: 'Internal Error on Proposal {0}',
+    errors: [{
+      name: 'invalidAddress',
+      message: 'Invalid Address'
+    },
+    {
+      name: 'invalidP2SHAddress',
+      message: 'Invalid P2SHAddress'
+    },
+    {
+      name: 'invalidPayment',
```

Comment:
Same note on plural vs singular, since Trigger objects only have a list of amounts which is a pipe-delimited string.